alignak/external_command.py

```diff
@@ -2981,7 +2981,7 @@ def process_host_check_result(self, host, status_code, plugin_output):
             chk.get_outputs(plugin_output, host.max_plugins_output_length)
             chk.status = 'waitconsume'
             chk.check_time = self.current_timestamp  # we are using the external command timestamps
-            # Set the corresponding host's check_type to passive=1
+            # Set the corresponding host's check type to passive
```

Comment:
Why? We set the host's check type to passive ... it is clear enough: passive is really the type of the host check...